packages/lego-bricks/src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,93 @@
+import cx from 'classnames';
+import { useEffect, useRef, useState } from 'react';
+import styles from './Accordion.module.css';
+
+type TriggerProps = {
+  onClick: () => void;
+  disabled: boolean;
+  open: boolean;
+  rotateClassName: string;
+};
+
+type Props = {
+  disabled?: boolean;
+  defaultOpen?: boolean;
+  animated?: boolean;
+  wrapperClassName?: string;
+  triggerPosition?: 'over' | 'under';
```

Comment:
I feel like it's more common to say "top" and "bottom" when referring to positions, but this is fine

Reply:
Absolutely agree